dace/codegen/targets/intel_fpga.py

```diff
@@ -67,12 +67,15 @@ def cmake_options():
         target_board = Config.get("compiler", "intel_fpga", "board")
         enable_debugging = ("ON" if Config.get_bool(
             "compiler", "intel_fpga", "enable_debugging") else "OFF")
+        autobuild = ("ON" if Config.get_bool(
```

Comment:
So are you disabling autobuild by default?

Reply:
Yes. Do you think this makes sense? It will still autobuild emulation kernels, just not builds that take many hours.